src/main/scala/org/apache/spark/deploy/armada/Config.scala

```diff
@@ -120,4 +133,91 @@ private[spark] object Config {
       .stringConf
       .checkValue(serviceNamePrefixValidator, "Service name prefix must adhere to rfc 1035 label names.")
       .createWithDefaultString("armada-spark-driver-")
+
+  private val invalidLabelListErrorMessage =
+    "Must be a comma-separated list of valid Kubernetes labels (each as key=value)"
+
+  val ARMADA_SPARK_GLOBAL_LABELS: ConfigEntry[String] =
+    ConfigBuilder("spark.armada.global.labels")
+      .doc("A comma separated list of kubernetes labels (in key=value format) to be added to all " +
+        "both the driver and executor pods.")
+      .stringConf
+      .checkValue(selectorsValidator, invalidLabelListErrorMessage)
```

Comment:
Why are there two different validators, `selectorsValidator` and `k8sLabelListValidator`?